Automattic/jetpack
# Revise Jetpack connection agreement text to comply with our User Agreement

Fixes # 1202858161751496-as-1203651563192009/f

## Proposed changes:

* Update the connection agreement text for all plugins to show above the button to comply with the guidelines PCYsg-9m2-p2

### Other information:


## Jetpack product discussion


p1HpG7-_kep-p2

## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Spin up a test site
* Jetpack Plugin
   * Build Jetpack plugin by running `jetpack build plugins/jetpack` 
   * Install and Navigate to the Jetpack page (`wp-admin/admin.php?page=jetpack#/`)
   * Confirm that Terms and condition text is displayed above the button 
   * ![Screenshot 2023-01-17 at 2 32 44 PM](https://user-images.githubusercontent.com/2027003/212854939-66a72d98-1edc-4942-b403-1dccd93f9d30.png)
* Jetpack ValutPress Backup plugin 
   * Build Jetpack Backup plugin by running `jetpack build plugins/backup` 
   * Install and Navigate to Jetpack Backup page (`wp-admin/admin.php?page=jetpack-backup`)
   * Confirm that Terms and condition text is displayed above the button
   * ![Screenshot 2023-01-17 at 2 37 00 PM](https://user-images.githubusercontent.com/2027003/212856670-001676ad-197c-4220-b17d-b8497ed87c4f.png)
* Jetpack Boost Plugin 
   *  Build Jetpack Boost plugin by running `jetpack build plugins/boost` 
   *  Install and Navigate to Jetpack Boost page (`wp-admin/admin.php?page=jetpack-boost`)
   * Confirm that Terms and condition text is displayed above the button
   * ![Screenshot 2023-01-17 at 2 43 27 PM](https://user-images.githubusercontent.com/2027003/212857185-60743433-977e-4af3-99c8-5b3f1d3e3674.png)
* Jetpack Social Plugin 
   *  Build Jetpack Boost plugin by running `jetpack build plugins/social` 
   *  Install and Navigate to Jetpack Boost page (`wp-admin/admin.php?page=jetpack-social`)
   * Confirm that Terms and condition text is displayed above the button
   * ![Screenshot 2023-01-17 at 2 45 09 PM](https://user-images.githubusercontent.com/2027003/212857550-af029cb1-5965-4af6-ab5d-46f1c6edaa20.png)
* My Jetpack - Connect your site screen 
  * Build Jetpack plugin by running `jetpack build plugins/jetpack` 
  * Install and Navigate to the My Jetpack page (`wp-admin/admin.php?page=my-jetpack`) and ensure you still haven't connected your site.
  * You will see the message as below and click on the button
  * ![Screenshot 2023-01-17 at 2 54 27 PM](https://user-images.githubusercontent.com/2027003/212863125-5caa6453-6ef5-40ff-8815-3736fbf1176d.png)

  * On the connection screen confirm that Terms and condition text is displayed above the button
  * ![Screenshot 2023-01-17 at 3 08 19 PM](https://user-images.githubusercontent.com/2027003/212862940-01a3c127-16dc-4749-9e8e-b78325e3683c.png)




---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203651563192009
